vagrantcommand.go

```diff
@@ -15,15 +15,25 @@ limitations under the License.
 
 package vagrantssh
 
+import "os"
 import "os/exec"
 
 // VagrantCommand is a command that is run on a vagrant node
 type VagrantCommand struct {
 	ContivNodes int
+	Env        string
 }
 
 func (c *VagrantCommand) getCmd(cmd string, args ...string) *exec.Cmd {
-	return exec.Command("vagrant", append([]string{cmd}, args...)...)
+	newArgs := append([]string{cmd}, args...)
+	osCmd := exec.Command("vagrant", newArgs...)
+	osCmd.Env = os.Environ()
```

Comment:
I really do not like the idea of just taking the system environment and running with it. We have methods for shipping the environment around; let's use them.